reactioncommerce/reaction
# [WIP] 1445 Add to Cart Alert Behavior

Resolves [1445](https://github.com/reactioncommerce/reaction/issues/1445)

***Test***
- Clone this branch
- Start the application
- Select product and click add to cart rapidly and notice the button becomes deactivated until cart alert is hidden
